### PR TITLE
Add R client news / update docs

### PR DESCRIPTION
**Pull request by m-mohr**
Did I capture all the main news in the client and are all description correct?

### CODE REVIEW

File: news/2022-05-12-r-client-v120.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+---
+news: true
+title:  R Client v1.2.0 has been released
+date:   2022-05-12
+author: Matthias Mohr, Florian Lahn
+---
+
+Today, we released a new version 1.2.0 of the openEO R Client, which features a couple of [improvements and bug fixes](https://github.com/Open-EO/openeo-r-client/blob/master/NEWS.md#version-120).
```
Comment (flahn) [inline news/2022-05-12-r-client-v120.md:8]: We can just link to the release version which directly contains the changes introduced with this proper release:
https://github.com/Open-EO/openeo-r-client/releases/tag/v1.2.0

**Review by flahn — COMMENTED** [review]
We can either link directly to the release version for changes or point to the NEWS.md. I'm fine with either

File: news/2022-05-12-r-client-v120.md (hunk 1 above)
Comment (m-mohr) [reply news/2022-05-12-r-client-v120.md:8]: I have no preference.

Comment (m-mohr) [reply news/2022-05-12-r-client-v120.md:8]:
```suggestion
Today, we released a new version 1.2.0 of the openEO R Client, which features a couple of [improvements and bug fixes](https://github.com/Open-EO/openeo-r-client/releases/tag/v1.2.0).
```

**Pull request merged**